Sources/PixelKit/PixelKit.swift

```diff
@@ -356,6 +365,23 @@ public final class PixelKit {
         fireRequestWrapper(pixelName + "_count", headers, newParams, allowedQueryReservedCharacters, true, .dailyAndCount, onComplete)
     }
 
+    private func handleDailyAndStandard(_ pixelName: String,
+                                        _ headers: [String: String],
+                                        _ newParams: [String: String],
+                                        _ allowedQueryReservedCharacters: CharacterSet?,
+                                        _ onComplete: @escaping CompletionBlock) {
+        reportErrorIf(pixel: pixelName, endsWith: "_u")
+        reportErrorIf(pixel: pixelName, endsWith: "_daily") // Because is added automatically
+        if !pixelHasBeenFiredToday(pixelName) {
+            fireRequestWrapper(pixelName + "_daily", headers, newParams, allowedQueryReservedCharacters, true, .dailyAndCount, onComplete)
+            updatePixelLastFireDate(pixelName: pixelName)
+        } else {
+            printDebugInfo(pixelName: pixelName + "_daily", frequency: .dailyAndCount, parameters: newParams, skipped: true)
+        }
+
+        fireRequestWrapper(pixelName, headers, newParams, allowedQueryReservedCharacters, true, .dailyAndCount, onComplete)
+    }
+
```

Comment:
This code largely mimics `handleDailyAndCount` with the exception of the last line where it fires the pixel with its standard name.